x-pack/plugins/security_solution/public/timelines/components/timeline/expandable_event/index.tsx

```diff
@@ -95,19 +109,26 @@ export const ExpandableEvent = React.memo<Props>(
       return <EuiTextColor color="subdued">{i18n.EVENT_DETAILS_PLACEHOLDER}</EuiTextColor>;
     }
 
-    if (loading) {
-      return <EuiLoadingContent lines={10} />;
-    }
-
     return (
-      <ExpandableDetails>
-        <LazyAccordion
-          id={`timeline-${timelineId}-row-${event.eventId}`}
-          renderExpandedContent={handleRenderExpandedContent}
-          forceExpand={!!event.eventId && !loading}
-          paddingSize="none"
-        />
-      </ExpandableDetails>
+      <>
+        <EuiFlyoutHeader hasBorder>
```

Comment:
can we revert back to the previous logic, to make sure we don't sure `EuiFlyoutHeader` and `EuiFlyoutBody` inside Timeline column as it itroduces too much of the spacing which reduces amount of visible content in the timeline
![Uploading image.png…]()